cvat-ui/src/components/create-task-page/basic-configuration-form.tsx

```diff
@@ -4,15 +4,19 @@
 
 import React, { RefObject } from 'react';
 import Input from 'antd/lib/input';
+import Text from 'antd/lib/typography/Text';
+import Tooltip from 'antd/lib/tooltip';
 import Form, { FormInstance } from 'antd/lib/form';
-import { Store } from 'antd/lib/form/interface';
 
 export interface BaseConfiguration {
     name: string;
 }
 
 interface Props {
-    onSubmit(values: BaseConfiguration): void;
+    onChange(values: BaseConfiguration): void;
+    isMultiTask: boolean;
+    exampleMultiTaskName?: string;
+    defaultValue?: string;
```

Comment:
I see the following logic: 
`` const defaultValue = isMultiTask ? '{{file_name}}' : '';``

isMultiTask is also a prop of this component, so, let's put this logic inside and remove this extra prop.